src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -73,12 +103,12 @@ class WorkflowView extends Component {
   }
 
   async componentDidMount() {
-    const { workspaceNamespace, workspaceName, workflowNamespace, workflowName } = this.props
-
-    const config = await Rawls.workspace(workspaceNamespace, workspaceName).methodConfigs.get(workflowNamespace, workflowName)
+    const config = await this.rawlsMethodConfig.get()
```

Comment:
Benefit to my ajax endpoint structuring: you can store the nested object and avoid boilerplate